src/Python/Core/py3d_registration.cpp

```diff
@@ -326,4 +364,9 @@ void pybind_registration_methods(py::module &m)
             "Function for computing information matrix from RegistrationResult",
             "source"_a, "target"_a, "max_correspondence_distance"_a,
             "transformation_result"_a);
+    m.def("texture_map_optimization",
+			&TextureMapOptimization,
+			"Function for optimizing texture mapping for reconstructed scenes",
+			"mesh"_a, "imgs_rgbd"_a, "camera"_a,
+            "option"_a = TextureMapOptmizationOption());
```

Comment:
I put texture_map_optimization in registration module. Not quite sure this is the right place.